presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -296,4 +370,29 @@ private long calculateTotalRows(int scaleBase, double scaleFactor)
         }
         return (long) totalRows;
     }
+
+    public static Type getNumericType(TypeSignature numericTypeSignature)
+    {
+        if (numericTypeSignature.getBase().equals(DECIMAL)) {
```

Comment:
I updated https://github.com/martint/presto/tree/pr4810 to remove that commit. We're cutting a release soon, so I'll merge it as soon as that's done.